### PR TITLE
fix: CLAW reference

### PR DESCRIPTION
**Pull request by a-b**
Admin permissions was not properly applied due to case sensitivity?

### CODE REVIEW

**Comment by beyhan** [comment]
@a-b I checked that the team [temp-ari-cli-admins](https://github.com/orgs/cloudfoundry/teams/temp-ari-cli-admins) has admin rights in the CALW repo. Do you miss something?

**Comment by a-b** [comment]
hi @beyhan , yes unfortunately admin rights configured in cff are not propagated to the repo

```
$ gh api repos/cloudfoundry/CLAW | jq .permissions   
{
  "admin": false,
  "maintain": false,
  "push": false,
  "triage": false,
  "pull": true
}
```
as a result, we can't even push. Our theory this is because of case sensitivity. The repo name is [https://github.com/cloudfoundry/**CLAW**](https://github.com/cloudfoundry/CLAW)

File: org/cloudfoundry.yml [hunk 1]
```diff
@@ -2388,7 +2388,7 @@ orgs:
         members: []
         privacy: closed
         repos:
-          claw: admin
+          CLAW: admin
```
Comment (a-b) [inline org/cloudfoundry.yml:2391]: Our working theory is that case sensitivity is breaking cloudfoundry/community synchronization

**Review by Gerg — APPROVED** [review]

**Comment by a-b** [comment]
Thanks for the approval, @Gerg!

**Comment by a-b** [comment]
@Gerg for some reason we lost control of this repo overnight

`gh api repos/cloudfoundry/CLAW | jq .permissions`
```
{
  "admin": false,
  "maintain": false,
  "push": false,
  "triage": false,
  "pull": true
}
```

**Review by stephanme — APPROVED** [review]

**Comment by stephanme** [comment]
Fixes spelling. TOC approved the temporary admin team in #441.

**Pull request merged**